### PR TITLE
Add instructions to install preview releases

### LINKED ISSUE #2573: Installing powershell preview builds for mac and linux has changed
Issue Details
-----------------

For linux, the change is only for preview build the package name has changed to `powershell-preview`

For mac, the change is to install a preview build you should do the following:

```sh
brew tap caskroom/versions
brew cask install powershell-preview
```

Version(s) of document impacted
------------------------------
- [x] Impacts 6.1 document
- [ ] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not selecting all version of documents
----------------------------------------------------
- [x] The documented feature was introduced in selected version of PowerShell
- [ ] This issue only shows up in selected version of the document

### PR DESCRIPTION
**Pull request by TravisEz13**
Add instructions to install preview releases

Also added notes about what distro's are supported experimentally

fixes #2573

### CODE REVIEW

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md [hunk 1]
```diff
@@ -25,6 +25,22 @@ Once the package is installed, run `pwsh` from a terminal.
...
+
+When installing a PowerShell Core Preview release for Linux via a Package Repository,
+the package name changes from `powershell` to `powershell-preview`.
+
+Installing via direct download does not change, other than the file name.
+
+Here is a table of the commands to install the stable and preview packages using the various package managers:
+
+|Stable Command | Preview Command |
+|---------------|-----------------|
```
Comment (daxian-dbw) [inline reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: How about adding a column in the left about the distros?

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md [hunk 2]
```diff
@@ -127,7 +143,7 @@ sudo apt-get remove powershell
 
 ## Ubuntu 17.10
 
-> Note: Support for Ubuntu 18.04 was added after `6.1.0-preview.2`
+> Please note, Support for Ubuntu 18.04 was added after `6.1.0-preview.2`
```
Comment (daxian-dbw) [inline reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: > Ubuntu 18.04

Should be `17.10`

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md](https://github.com/TravisEz13/PowerShell-Docs/blob/linux_preview/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-3.0&branch=pr-en-us-2617) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-4.0&branch=pr-en-us-2617) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-5.0&branch=pr-en-us-2617) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-5.1&branch=pr-en-us-2617) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-6&branch=pr-en-us-2617) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C25%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806252303358463-2617%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=X04DbsN8N5X6C%2FET77qZNb%2B3FIk2Zy58YDvMNYspx6o%3D&st=2018-06-25T23%3A09%3A47Z&se=2018-07-26T23%3A14%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md](https://github.com/TravisEz13/PowerShell-Docs/blob/linux_preview/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-3.0&branch=pr-en-us-2617) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-4.0&branch=pr-en-us-2617) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-5.0&branch=pr-en-us-2617) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-5.1&branch=pr-en-us-2617) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-6&branch=pr-en-us-2617) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C25%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806252325309138-2617%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Tt2UoaHj43bzfldBZFC%2F5a6wcAOIgbFvJ%2BgD0NVT4Pk%3D&st=2018-06-25T23%3A31%3A27Z&se=2018-07-26T23%3A36%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md](https://github.com/TravisEz13/PowerShell-Docs/blob/linux_preview/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-3.0&branch=pr-en-us-2617) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-4.0&branch=pr-en-us-2617) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-5.0&branch=pr-en-us-2617) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-5.1&branch=pr-en-us-2617) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-6&branch=pr-en-us-2617) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C25%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806252342510621-2617%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=DDN%2BPOltshyLJyXSBEf69PeXWkCkcREoyQy7wuU1M4E%3D&st=2018-06-25T23%3A49%3A21Z&se=2018-07-26T23%3A54%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sdwheeler** [comment]
Holding for format changes. I will do this tomorrow morning.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md](https://github.com/TravisEz13/PowerShell-Docs/blob/linux_preview/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-3.0&branch=pr-en-us-2617) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-4.0&branch=pr-en-us-2617) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-5.0&branch=pr-en-us-2617) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-5.1&branch=pr-en-us-2617) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-6&branch=pr-en-us-2617) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C26%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806261505464049-2617%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=yAoozx2HYdUlv9lDCRpRByL1spjwrTVG7k61fnCFuBM%3D&st=2018-06-26T15%3A12%3A27Z&se=2018-07-27T15%3A17%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**